src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
```

Comment:
i'm only partially familiar with the async keyword in JS, could you explain to me why it's here, when the other app.get() invocations do not seem to use async? this seems by it's nature a function that should be synchronous, but i could be misunderstanding something.

Reply:
From my understanding, async is useful if you're trying to await something in your code. In my case, I'm awaiting the data from the Nearby search API call.  All of the app.get() invocations use async.